frontend/src/routes/index.tsx

```diff
@@ -0,0 +1,25 @@
+import loadable from '@loadable/component'
+import React from 'react'
+import { Route, Switch } from 'react-router-dom'
+
+export enum RouteName {
+    HOME = '/'
+}
+
+
+const Home = loadable(() => import('../components/pages/home'), {
+    fallback: <div>Loading...</div>
+})
+
+export const Routes: React.FC = () => {
+    return (
+        <React.Fragment>
+            <Switch>
+                <Route path={RouteName.HOME}>
```

Comment:
`<Route path={RouteName.HOME}>` causes every subsequent route containing "/" to route to the HOME component, not sure if this is desired, a fix could be `<Route exact path={RouteName.HOME}>`

Reply:
@peterghrong yes It is not desired with the reasoning you stated. But because this is just scaffolding, and we won't be having a `Home` page anyways meaning it is temporary I did not bother to add it. This pr was made to lay down the general structure so most of it is just filler code atm.

Reply:
@peterghrong Also, with correct placement of `Route` components, you wouldn't necessarily have to toggle the `exact` property to avoid the effect of every subsequent route routing to `Home`.

Reply:
Understood, thank you !!